### ISSUE TITLE
Add support for raw encryption for crypto key in KMS module.

### ISSUE THREAD

**Issue by romanini-ciandt** [terraform-google-modules/terraform-google-kms#130]
### TL;DR

[google_kms_crypto_key resource](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/kms_crypto_key) supports [import_only](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/kms_crypto_key#import_only) and [skip_initial_version_creation](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/kms_crypto_key#skip_initial_version_creation) input variables, but it is not possible to achieve the same crypto key creation (if set both variables mentioned as `true`) using the KMS module. The KMS module doesn't have these inputs options.

Usually, this use case will happen when the `purpose` attribute is `RAW_ENCRYPT_DECRYPT` - which is not mentioned as a possible value in [module's inputs documentation](https://registry.terraform.io/modules/terraform-google-modules/kms/google/2.3.0#inputs).

### Terraform Resources

```markdown
google_kms_crypto_key
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/kms_crypto_key#
```
